lambda_function.py

```diff
@@ -44,18 +40,23 @@ def lambda_handler(event, lambda_context):
                 and (k in message["purpose"]["params"] for k in ("platform", "id"))
             ):
 
-                # parsing through each user ID and its respective valid flag
-                for each in message["user"]["values"]:
+                # extract all ID's in the array
+                user_values = message["user"]["values"]
+                user_values_length = len(user_values)
```

Comment:
I dont think this variable is needed really, can just do with `len(user_values)`